src/common/components/LoadingButton/index.tsx

```diff
@@ -4,25 +4,23 @@ import styled from 'styled-components';
 
 const BootstrapButton: FC<ButtonProps> = ({ children, ...rest }) => <Button {...rest}>{children}</Button>;
 
-type LoadingButtonProps = {
-  disabled?: boolean;
-  loading?: boolean;
-};
+type LoadingButtonProps = ButtonProps & { loading: boolean };
 
 const StyledButton = styled(BootstrapButton)`
   color: ${props => props.theme.buttons.submitTextColor};
   background-color: ${props => props.theme.buttons.submitBackgroundColor};
   border-color: ${props => props.theme.buttons.submitBorderColor};
   padding: 0.5rem 1rem;
+  // width: 40%;
```

Comment:
Do we need this?

Reply:
I kept it here on purpose because it will be part of the button/styling PR that follows this, and I wanted to remember the styles we had before that worked.